forge-ai/src/main/java/forge/ai/ability/DrawAi.java

```diff
@@ -547,4 +549,53 @@ public boolean confirmAction(Player player, SpellAbility sa, PlayerActionConfirm
         // except it has Laboratory Maniac
         return player.isCardInPlay("Laboratory Maniac");
     }
+
+    @Override
+    public boolean willPayUnlessCost(SpellAbility sa, Player payer, Cost cost, boolean alreadyPaid, FCollectionView<Player> payers) {
+        final Card host = sa.getHostCard();
+
+        // TODO better logic for Indulgent Tormentor
+        final String aiLogic = sa.getParam("UnlessAI");
+        if ("Never".equals(aiLogic)) { return false; }
+
+        if (alreadyPaid && payers.size() > 1) {
+            return false;
+        }
+
+        if ("LowPriority".equals(aiLogic) && MyRandom.getRandom().nextInt(100) < 67) {
+            return false;
+        }
+
+        // Risk Factor Effects
+        for (Player p : AbilityUtils.getDefinedPlayers(host, sa.getParam("Defined"), sa)) {
+            if (p.isOpponentOf(payer)) {
+                if (!p.canDraw()) {
+                    return false;
+                }
+                if (cost.hasSpecificCostType(CostDamage.class)) {
```

Comment:
all these extra parts checks don't make sense to me

they should already be covered by the `super` anyway

Reply:
The generic super doesn't check for Damage in comparison to the amount of cards being drawn